### PR TITLE
nsupdate_web: add nsupdate_web branch parameter

### PR DESCRIPTION
**Pull request by kshtsk**

### CODE REVIEW

**Comment by kshtsk** [comment]
@djgalloway could you please review

File: roles/nsupdate_web/defaults/main.yml [hunk 1]
```diff
@@ -5,8 +5,9 @@ nsupdate_web_port: "8080"
 nsupdate_web_ttl: "60"
 virtualenv_path: "~/venv"
 python_version: "python3"
-repo_url: "https://github.com/zmc/nsupdate-web.git"
-repo_path: "/home/{{ nsupdate_web_user }}/nsupdate_web"
+nsupdate_web_repo: "https://github.com/zmc/nsupdate-web.git"
```
Comment (djgalloway) [inline roles/nsupdate_web/defaults/main.yml]: @zmc changed ownership (or copied) the nsupdate-web repo to https://github.com/ceph/nsupdate-web.  Can you change this line to reflect that please?

Comment (kshtsk) [reply roles/nsupdate_web/defaults/main.yml]: @djgalloway I'm aware of it, but I would like to do this in separate PR

Comment (djgalloway) [reply roles/nsupdate_web/defaults/main.yml]: Why not this PR?

Comment (kshtsk) [reply roles/nsupdate_web/defaults/main.yml]: > Why not this PR?

The PR created before @zmc created the fork, and I've not tested it.

Comment (kshtsk) [reply roles/nsupdate_web/defaults/main.yml]: I can submit a commit for this later here, but it will take some time before I test in my environment.

Comment (djgalloway) [reply roles/nsupdate_web/defaults/main.yml]: Master branches are the same.  https://github.com/zmc/nsupdate-web/compare/master...ceph:master

I know I'm nit-picking but I'd rather not merge technical debt no matter how small.

Comment (kshtsk) [reply roles/nsupdate_web/defaults/main.yml]: @djgalloway here you are

Comment (djgalloway) [reply roles/nsupdate_web/defaults/main.yml]: Thank you!

**Pull request merged**